wrangler.toml

```diff
@@ -22,7 +22,7 @@ services = [
 ]
 
 [triggers]
-crons = ["0 */6 * * *","0 */11 * * *","*/20 * * * *","*/30 * * * *" ]
+crons = ["0 */11 * * *","*/20 * * * *" ]
```

Comment:
11 hr cron? or should it be 12?
Any reason why 11 hours?

Reply:
Earlier we used 11 & 6, but kept it as 11 for now
Can make it 12, can change it as per the need